.github/workflows/docker.yml

```
@@ -0,0 +1,39 @@
+name: Publish Docker image
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - ./Gemfile
+      - ./Dockerfile
+      - ./scripts/entrypoint.sh
+      - ./github/workflows/docker.yml
+jobs:
+  push_to_registry:
+    name: Publish Docker Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Setup Docker QEMU
+        uses: docker/setup-qemu-action@v1.2.0
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1.5.1
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1.10.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1.10.0
+        with:
+          registry: ghcr.io
+          username: ${{ GITHUB.ACTOR }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v2.7.0
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          push: true
+          tags: ${{ GITHUB.REPOSITORY }}:latest
```

```suggestion
          tags: |
            ${{ GITHUB.REPOSITORY }}:latest
            ghcr.io/${{ GITHUB.REPOSITORY }}:latest
```

https://github.com/docker/build-push-action/blob/master/docs/advanced/push-multi-registries.md